docs/team/johndoe.adoc

```diff
@@ -61,9 +61,6 @@ include::../UserGuide.adoc[tag=dataencryption]
 
 include::../DeveloperGuide.adoc[tag=undoredo]
 
-include::../DeveloperGuide.adoc[tag=dataencryption]
```

Comment:
Why are you removing this as part of this PR?

Reply:
I thought I made this change in another branch. I've added it back in.